Extensions/tweaks.js

```diff
@@ -99,6 +99,12 @@ XKit.extensions.tweaks = new Object({
 			text: "User Interface tweaks",
 			type: "separator",
 		},
+		"fix_activity_feed": {
+			text: "Re-slim the Activity feed and highlight notes from mutuals",
```

Comment:
Mobile still highlights mutuals as far as I know, desktop's highlighting got changed to highlighting just the URL in a dark blue http://april.sylph.tech/post/161038068793